src/fosslight_scanner/_run_compare.py

```diff
@@ -104,6 +104,13 @@ def write_result_html(output_file, compared_result, before_yaml, after_yaml):
                         tr.insert(i, td)
                     table_html.insert(row, tr)
                     row += 1
+                if row >= 102:
```

Comment:
Considering the minimum row change, it is recommended to declare and use 
`MIN_ROW_NUM = 100`
`row >= MIN_ROW_NUM +2`